package.json

```diff
@@ -17,14 +17,14 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^11.1.2",
-    "@angular/common": "^11.1.2",
-    "@angular/compiler": "^11.1.2",
-    "@angular/core": "^11.1.2",
-    "@angular/forms": "^11.1.2",
-    "@angular/platform-browser": "^11.1.2",
-    "@angular/platform-browser-dynamic": "^11.1.2",
-    "@angular/router": "^11.1.2",
+    "@angular/animations": "^12.2.1",
+    "@angular/common": "^12.2.1",
+    "@angular/compiler": "^12.2.1",
+    "@angular/core": "^12.2.1",
+    "@angular/forms": "^12.2.1",
+    "@angular/platform-browser": "^12.2.1",
+    "@angular/platform-browser-dynamic": "^12.2.1",
+    "@angular/router": "^12.2.1",
```

Comment:
what about zone.js? does it need to be upgraded?

Reply:
No, I think we're already on at latest one https://www.npmjs.com/package/zone.js?activeTab=versions